nise/generators/ocp/ocp_generator.py

```diff
@@ -737,8 +757,9 @@ def _randomize_ros_ocp_line_values(self, pod_in):
     def _update_ros_ocp_pod_data(self, row, start, end, **kwargs):
         """Update data with generator specific data."""
         pod = kwargs.get("pod")
-        if pod:
-            pod = self._randomize_ros_ocp_line_values(pod)
+        if not self.constant_values_ros_ocp:
```

Comment:
any reason not to use `if pod and not  self.constant_values_ros_ocp:` ?

Reply:
There was a strange behavior when i used the above and operator. When I passed flag as true it didnt generate constant values but when flag was false then it generated constant values
So i decided to go with above approach

Reply:
are you sure you didn't miss the "not" in the condition?

Reply:
ah sorry I had just used `if pod and  self.constant_values_ros_ocp:`